codespell_lib/data/dictionary.txt

```diff
@@ -30197,6 +30262,7 @@ protruberances->protuberances
 prouncements->pronouncements
 provacative->provocative
 provate->private, provide,
+provde->provide
```

Comment:
It could also be `proved` in some cases, couldn't it?

Also:
```
provdes->provides
```

Reply:
Yes, there are examples of `provde->proved`